### DISCUSSION TITLE
Museum self-hosted - Replication

### DISCUSSION THREAD

**Discussion by PaWyss** [ente-io/ente#2547] [Q&A] [answered]
Hi folks,

I spend the last couple of days with self-hosting Ente and I'm love it. Everything is running so far and secure behind a reverse proxy. The last thing I try now is to replicate to multiple S3 buckets, local (three Minio instance's: one on the Ente environment, two on different TrueNas.

Has anyone done something like that? If always the error: 
"WARN[0000]main.go:809 setupAndStartBackgroundJobs Could not start replication v3: replication.worker-url was not defined "

I tested them separatly, and that was working (without replication).

I have replication on:
replication:
  enabled: true

Ans I thought the "worker-url" is not mandatory?

Or maybe just a replication from one Minio to another Minio would be easier for backup ;)

**Comment by mnvr** [comment, marked as answer]
> "worker-url" is not mandatory?

Correct. This change was not in the latest docker image, it would only work if you were building from source.

I've just published a new docker image, so it should work with the docker image too now, just pull the latest.

**Reply by PaWyss** [reply]
Wohooo, and it just works! Great, many thanks!
